### BIG differences, this is diff of wrong files.
### CLUS "Type Specifier NOT"
### and CLHS topic selection http://clhs.lisp.se/Body/a_not.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:not.txt	2017-04-27 21:39:48.000000000 +0100
+++ saved-pages-clus/cl:types:not.txt	2017-04-27 21:39:48.000000000 +0100
@@ -1,4 +1,34 @@
-NOT
-Please select which reference to NOT you intended
+Type Specifier NOT

-FunctionType Specifier
\ No newline at end of file
+Compound Type Specifier Kind
+Combining.
+Compound Type Specifier Syntax
+not typespec
+Compound Type Specifier Arguments
+typespec - a type specifier.
+Compound Type Specifier Description
+This denotes the set of all objects that are not of the type typespec.
+The argument is required, and cannot be *.
+The symbol not is not valid as a type specifier.
+\
\ No newline at end of file
